BookEcommerce.Models/DAL/Interfaces/IProductVariantRepository.cs

```diff
@@ -9,5 +9,7 @@ namespace BookEcommerce.Models.DAL.Interfaces
 {
     public interface IProductVariantRepository : IRepository<ProductVariant>
     {
+        Task<List<ProductVariant>> GetProductVariantByIdProduct(string? idProduct);
```

Comment:
productGuid? could we update it to using guid instead of string

Reply:
I got it